src/main/java/lc/CaptchaTypes.java

```diff
@@ -0,0 +1,117 @@
+package lc;
+
+import javax.imageio.ImageIO;
+import javax.imageio.stream.FileImageOutputStream;
+import javax.imageio.stream.ImageOutputStream;
+import java.awt.*;
+import java.awt.font.TextLayout;
+import java.awt.image.BufferedImage;
+import java.awt.image.ConvolveOp;
+import java.awt.image.Kernel;
+import java.io.IOException;
+import java.io.File;
+import java.util.Random;
+import java.io.ByteArrayOutputStream;
+
+public class CaptchaTypes{
+
+    private String secret = "";
+
+    private void setRenderingHints(Graphics2D g2d){
+        g2d.setRenderingHint(RenderingHints.KEY_TEXT_ANTIALIASING,
+                RenderingHints.VALUE_TEXT_ANTIALIAS_ON);
+        g2d.setRenderingHint(RenderingHints.KEY_FRACTIONALMETRICS,
+                RenderingHints.VALUE_FRACTIONALMETRICS_ON);
+    }
+
+    private BufferedImage charToImg(String text){
+        BufferedImage img = new BufferedImage(250, 100, BufferedImage.TYPE_INT_RGB);
+        Font font = new Font("Bradley Hand", Font.ROMAN_BASELINE, 48);
+        Graphics2D graphics2D = img.createGraphics();
+        graphics2D.setFont(font);
+        graphics2D.setColor(new Color((int)(Math.random() * 0x1000000)));
+        graphics2D.drawString( text , 45, 45);
+        graphics2D.dispose();
+        return img;
+    }
+
+    public byte[] type_1(String captchaText){
```

Comment:
Separate each type of captcha into separate classes that implement `CaptchaProvider` interface.

Reply:
Okay!